ecocode-rules-specifications/README.md

```diff
@@ -1,3 +1,4 @@
+git
```

Comment:
I don't understand this added line in this file ... why ?

Reply:
Oh sorry, it's a mistake.
I removed it